### ISSUE TITLE
Using accumulo config-upgrade to perform an upgrade fails

### ISSUE THREAD

**Issue by EdColeman** [apache/accumulo#3041]
**Describe the bug**
If the stand-alone utility is used to upgrade from 1.10 to 2.1 the upgrade code fails when starting the cluster.

If the cluster is just started, the upgrade is successful.  It appear that the upgrade path on cluster start does not properly account for the zookeeper properties already being upgrade.  

If the cluster is started and the upgrade completes it looks like the upgrade is not re -riggered and there is not an issue.

To reproduce with uno:

Setup an accumulo 1.10 instance
uno stop
uno fetch an accumulo 2.1 branch
uno install accumulo --no-deps
uno start zookeeper
accumulo config-upgrade
uno accumulo start

On the cluster start:

```
2022-10-25T14:44:24,001 [util.ConfigPropertyUpgrader] INFO : Upgrading namespace +accumulo base path: /accumulo/0f34c3ad-0af4-4a0e-b5d2-458ecec36bb3/namespaces/+accumulo/conf
2022-10-25T14:44:24,004 [util.TransformToken] DEBUG: Failed to write transform token for /accumulo/0f34c3ad-0af4-4a0e-b5d2-458ecec36bb3/namespaces/+accumulo/conf/transform_token another process may have created one
org.apache.zookeeper.KeeperException$NoNodeException: KeeperErrorCode = NoNode for /accumulo/0f34c3ad-0af4-4a0e-b5d2-458ecec36bb3/namespaces/+accumulo/conf/transform_token
        at org.apache.zookeeper.KeeperException.create(KeeperException.java:118) ~[zookeeper-3.7.1.jar:3.7.1]
        at org.apache.zookeeper.KeeperException.create(KeeperException.java:54) ~[zookeeper-3.7.1.jar:3.7.1]
        at org.apache.zookeeper.ZooKeeper.create(ZooKeeper.java:1403) ~[zookeeper-3.7.1.jar:3.7.1]
        at org.apache.accumulo.core.fate.zookeeper.ZooReaderWriter.lambda$putEphemeralData$5(ZooReaderWriter.java:180) ~[accumulo-core-2.1.0-SNAPSHOT.jar:2.1.0-SNAPSHOT]
        at org.apache.accumulo.core.fate.zookeeper.ZooReader.retryLoopMutator(ZooReader.java:185) ~[accumulo-core-2.1.0-SNAPSHOT.jar:2.1.0-SNAPSHOT]
        at org.apache.accumulo.core.fate.zookeeper.ZooReader.retryLoop(ZooReader.java:164) ~[accumulo-core-2.1.0-SNAPSHOT.jar:2.1.0-SNAPSHOT]
        at org.apache.accumulo.core.fate.zookeeper.ZooReader.retryLoop(ZooReader.java:151) ~[accumulo-core-2.1.0-SNAPSHOT.jar:2.1.0-SNAPSHOT]
        at org.apache.accumulo.core.fate.zookeeper.ZooReaderWriter.putEphemeralData(ZooReaderWriter.java:180) ~[accumulo-core-2.1.0-SNAPSHOT.jar:2.1.0-SNAPSHOT]
        at org.apache.accumulo.server.conf.util.TransformToken.getTokenOwnership(TransformToken.java:99) ~[accumulo-server-base-2.1.0-SNAPSHOT.jar:2.1.0-SNAPSHOT]
        at org.apache.accumulo.server.conf.util.TransformToken.<init>(TransformToken.java:61) ~[accumulo-server-base-2.1.0-SNAPSHOT.jar:2.1.0-SNAPSHOT]
        at org.apache.accumulo.server.conf.util.TransformToken.createToken(TransformToken.java:80) ~[accumulo-server-base-2.1.0-SNAPSHOT.jar:2.1.0-SNAPSHOT]
        at org.apache.accumulo.server.conf.util.ConfigTransformer.transform(ConfigTransformer.java:110) ~[accumulo-server-base-2.1.0-SNAPSHOT.jar:2.1.0-SNAPSHOT]
        at org.apache.accumulo.server.conf.util.ConfigPropertyUpgrader.upgradeNamespaceProps(ConfigPropertyUpgrader.java:111) ~[accumulo-server-base-2.1.0-SNAPSHOT.jar:2.1.0-SNAPSHOT]
        at org.apache.accumulo.server.conf.util.ConfigPropertyUpgrader.doUpgrade(ConfigPropertyUpgrader.java:90) ~[accumulo-server-base-2.1.0-SNAPSHOT.jar:2.1.0-SNAPSHOT]
        at org.apache.accumulo.manager.upgrade.Upgrader9to10.upgradePropertyStorage(Upgrader9to10.java:209) ~[accumulo-manager-2.1.0-SNAPSHOT.jar:2.1.0-SNAPSHOT]
        at org.apache.accumulo.manager.upgrade.Upgrader9to10.upgradeZookeeper(Upgrader9to10.java:137) ~[accumulo-manager-2.1.0-SNAPSHOT.jar:2.1.0-SNAPSHOT]
        at org.apache.accumulo.manager.upgrade.UpgradeCoordinator.upgradeZookeeper(UpgradeCoordinator.java:156) ~[accumulo-manager-2.1.0-SNAPSHOT.jar:2.1.0-SNAPSHOT]
        at org.apache.accumulo.manager.Manager.setManagerState(Manager.java:302) ~[accumulo-manager-2.1.0-SNAPSHOT.jar:2.1.0-SNAPSHOT]
        at org.apache.accumulo.manager.Manager.getManagerLock(Manager.java:1534) ~[accumulo-manager-2.1.0-SNAPSHOT.jar:2.1.0-SNAPSHOT]

```

**Comment by DomGarguilo** [comment]
I was able to reproduce this with the listed steps and got the following error from the manager log:
```
2022-10-25T14:55:49,959 [util.ConfigPropertyUpgrader] INFO : Upgrading namespace +accumulo base path: /accumulo/400c837f-e752-4155-89c7-fde86e48e835/namespaces/+accumulo/conf
2022-10-25T14:55:49,966 [upgrade.UpgradeCoordinator] ERROR: FATAL: Error performing upgrade
java.lang.IllegalStateException: Failed to release transform lock for /accumulo/400c837f-e752-4155-89c7-fde86e48e835/namespaces/+accumulo/conf/transform_token
	at org.apache.accumulo.server.conf.util.TransformToken.releaseToken(TransformToken.java:165) ~[accumulo-server-base-2.1.0.jar:2.1.0]
	at org.apache.accumulo.server.conf.util.ConfigTransformer.transform(ConfigTransformer.java:186) ~[accumulo-server-base-2.1.0.jar:2.1.0]
	at org.apache.accumulo.server.conf.util.ConfigTransformer.transform(ConfigTransformer.java:111) ~[accumulo-server-base-2.1.0.jar:2.1.0]
	at org.apache.accumulo.server.conf.util.ConfigPropertyUpgrader.upgradeNamespaceProps(ConfigPropertyUpgrader.java:111) ~[accumulo-server-base-2.1.0.jar:2.1.0]
	at org.apache.accumulo.server.conf.util.ConfigPropertyUpgrader.doUpgrade(ConfigPropertyUpgrader.java:90) ~[accumulo-server-base-2.1.0.jar:2.1.0]
	at org.apache.accumulo.manager.upgrade.Upgrader9to10.upgradePropertyStorage(Upgrader9to10.java:209) ~[accumulo-manager-2.1.0.jar:2.1.0]
	at org.apache.accumulo.manager.upgrade.Upgrader9to10.upgradeZookeeper(Upgrader9to10.java:137) ~[accumulo-manager-2.1.0.jar:2.1.0]
	at org.apache.accumulo.manager.upgrade.UpgradeCoordinator.upgradeZookeeper(UpgradeCoordinator.java:156) ~[accumulo-manager-2.1.0.jar:2.1.0]
	at org.apache.accumulo.manager.Manager.setManagerState(Manager.java:302) ~[accumulo-manager-2.1.0.jar:2.1.0]
	at org.apache.accumulo.manager.Manager.getManagerLock(Manager.java:1534) ~[accumulo-manager-2.1.0.jar:2.1.0]
	at org.apache.accumulo.manager.Manager.run(Manager.java:1078) ~[accumulo-manager-2.1.0.jar:2.1.0]
	at org.apache.accumulo.core.trace.TraceWrappedRunnable.run(TraceWrappedRunnable.java:52) ~[accumulo-core-2.1.0.jar:2.1.0]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
Caused by: org.apache.zookeeper.KeeperException$NoNodeException: KeeperErrorCode = NoNode for /accumulo/400c837f-e752-4155-89c7-fde86e48e835/namespaces/+accumulo/conf/transform_token
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:118) ~[zookeeper-3.8.0.jar:3.8.0]
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:54) ~[zookeeper-3.8.0.jar:3.8.0]
	at org.apache.zookeeper.ZooKeeper.getData(ZooKeeper.java:1972) ~[zookeeper-3.8.0.jar:3.8.0]
	at org.apache.accumulo.core.fate.zookeeper.ZooReader.lambda$getData$1(ZooReader.java:84) ~[accumulo-core-2.1.0.jar:2.1.0]
	at org.apache.accumulo.core.fate.zookeeper.ZooReader.retryLoopMutator(ZooReader.java:185) ~[accumulo-core-2.1.0.jar:2.1.0]
	at org.apache.accumulo.core.fate.zookeeper.ZooReader.retryLoop(ZooReader.java:164) ~[accumulo-core-2.1.0.jar:2.1.0]
	at org.apache.accumulo.core.fate.zookeeper.ZooReader.retryLoop(ZooReader.java:151) ~[accumulo-core-2.1.0.jar:2.1.0]
	at org.apache.accumulo.core.fate.zookeeper.ZooReader.getData(ZooReader.java:84) ~[accumulo-core-2.1.0.jar:2.1.0]
	at org.apache.accumulo.server.conf.util.TransformToken.releaseToken(TransformToken.java:152) ~[accumulo-server-base-2.1.0.jar:2.1.0]
	... 12 more
```

**Comment by EdColeman** [comment]
Is that with the fix?

**Comment by DomGarguilo** [comment]
> Is that with the fix?

No, before the fix. See my comment on #3043

**Comment by EdColeman** [comment]
Resolved with #3043
